Extensions/Today/TodayViewController.swift

```diff
@@ -152,6 +137,11 @@ class TodayViewController: UIViewController, NCWidgetProviding, TodayWidgetAppea
     @objc func onPressNewPrivateTab(_ view: UIView) {
         openContainingApp("?private=true")
     }
+    
+    @objc func onPressClosePrivateTabs() {
```

Comment:
@noorhashem Do you plan on adding something here?

Reply:
@nbhasin2 yes, I'm planning to